### PR TITLE
Update to Hazelcast 3.11

### PR DESCRIPTION
**Pull request by googlielmo**
Use Hazelcast Client from official `hazelcast-all` OSGi bundle.
Client Bundle Activator by @emre-aydin was moved to this repo, so we don't have to modify upstream Hazelcast code.

### CODE REVIEW

File: hazelcast-dynacache-bundle/src/main/java/com/hazelcast/client/osgi/Activator.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2008-2016, Hazelcast, Inc. All Rights Reserved.
```
Comment (emre-aydin) [inline hazelcast-dynacache-bundle/src/main/java/com/hazelcast/client/osgi/Activator.java]: We should update the year.

**Review by emre-aydin — APPROVED** [review]
LGTM. Only a minor comment.

**Review by mesutcelik — APPROVED** [review]
Thanks @googlielmo ,
Can you update readme with the compatible version of websphere liberty if not done yet?

File: hazelcast-dynacache-bundle/src/main/java/com/hazelcast/client/osgi/Activator.java (hunk 1 above)
Comment (googlielmo) [reply hazelcast-dynacache-bundle/src/main/java/com/hazelcast/client/osgi/Activator.java]: done, thanks @emre-aydin

**Comment by googlielmo** [comment]
@mesutcelik @emre-aydin I addressed comments above

**Comment by mesutcelik** [comment]
👍

**Comment by googlielmo** [comment]
Fixes #1

**Pull request merged**